code/android-core-library/src/main/java/com/adobe/marketing/mobile/EventSource.java

```diff
@@ -37,4 +37,6 @@ private EventSource() {}
 	public static final String APPLICATION_CLOSE = "com.adobe.eventSource.applicationClose";
 	public static final String CONSENT_PREFERENCE = "consent:preferences";
 	public static final String UPDATE_CONSENT = "com.adobe.eventSource.updateConsent";
+	public static final String RESET_COMPLETE = "com.adobe.eventSource.resetComplete";
+	public static final String UPDATE_IDENTITY = "com.adobe.eventSource.updateIdentity";
```

Comment:
ERROR_RESPONSE_CONTENT = "com.adobe.eventSource.errorResponseContent" is missing, did you plan to keep that in Edge repo?